### ISSUE TITLE
validAliasCharacters field will be inserted in a "random" section

### ISSUE THREAD

**Issue by fritzmg** [ausi/contao-slug-backport#2]
In the `tl_page.php` you are using
```php
$GLOBALS['TL_DCA']['tl_page']['palettes']['root'] = str_replace(';{sitemap_legend', ',validAliasCharacters;{sitemap_legend', $GLOBALS['TL_DCA']['tl_page']['palettes']['root']);
```
This means the `validAliasCharacters` field will be inserted into whatever section comes before the `sitemap_legend` section. You are probably intending to insert the field into the _Global settings_ section - however, if an extension happens to insert another section _after_ __that__ section, the `validAliasCharacters` field will end up being in that section instead.

For instance, if you have the `news_categories` extension installed, the `validAliasCharacters` field will be part of the _News categories settings_ section of the palette.

**Comment by fritzmg** [comment]
Hm, I think you forgot a `use` statement.

**Comment by ausi** [comment]
Fixed in version 1.0.3
